Library/Formula/memo.rb

```diff
@@ -0,0 +1,17 @@
+class Memo < Formula
+  url "http://www.getmemo.org/memo-1.5.tar.gz"
+  homepage "http://www.getmemo.org"
+  sha1 "3e047b09e91d695f1767d1dd6d179732c07a5759"
+
+  def install
+    system "make", "PREFIX=#{prefix}"
+    system "make", "install"
+    bin.install "memo"
+    man1.install "memo.1"
```

Comment:
I might be mistaken here (or maybe just don't have that much experience with Homebrew yet), but when I was writing a formula for memo, calling `system "make", "install", "PREFIX=#{prefix}"`, the installation failed saying something like no files were installed, yet I could actually run memo. I took that as a sign that it was copying them to /usr/local/bin instead of memo's location in the Cellar.

Take a look at memo's install target (it's three lines): 
https://github.com/nrosvall/memo/blob/master/Makefile#L26-L28